frontend/src/css/modules/leaderboard.scss

```diff
@@ -50,3 +50,7 @@ span.badge.partial-evaluation {
   float: left;
   box-sizing: border-box;
 }
+
+.leaderboard-table {
+	width: 160%;
```

Comment:
@Alabhya268 this seems like a hacky fix. Can't we set leaderboard width to auto? Also, did you test how the leaderboard looks when there are less fields?

Reply:
@Ram81, by default table cells are taking width of longest word present in text. So making width auto will not have any effect on table cell width.

Reply:
@Alabhya268 no, this doesn't look good. We can't use this approach.